rest_framework/filters.py

```diff
@@ -282,7 +282,11 @@ class DjangoObjectPermissionsFilter(BaseFilterBackend):
     has read object level permissions.
     """
     def __init__(self):
-        assert guardian, 'Using DjangoObjectPermissionsFilter, but django-guardian is not installed'
+        try:
+            import guardian
+        except ImportError:
+            raise ImportError('Using DjangoObjectPermissionsFilter, but django-guardian is not installed')
```

Comment:
I think changing this to an ImportError instead of AssertionError is fine?!

Reply:
Probably use a `RuntimeError`.

Reply:
We could also include a comment linking to the issue. (As we do in `filter_queryset`)